### ISSUE TITLE
Identify new maintainer(s)?

### ISSUE THREAD

**Issue by leotac** [dmlc/decord#214]
Hi @zhreshold and contributors,

Given that @zhreshold stopped maintaining this library, would it be possible to identify some trustworthy maintainer(s) that could at least update builds / accept PRs to fix bugs?

**Comment by ruancomelli** [comment]
> (posting here to try to make some more noise)

This library is the best one I've ever used for this purpose, it would be lovely if @zhreshold and contributors could identify new maintainers.

**Comment by zhreshold** [comment]
Anyone interested? Currently I cannot allocate enough time for new features/bug fixes, but I could still review PRs and get involved in discussions.

**Comment by gaurav274** [comment]
I can help maintain this project. I've hacked it pretty much to fix multiple bugs. I'm working on another project ([EVA-DB](https://github.com/georgia-tech-db/Eva)) which is an AI-Relational Database System. I was hoping that I could merge my changes and help fix some bugs, and also want to use `decord` for the other project. We can talk, @zhreshold ?

**Comment by RichieHakim** [comment]
This repo remains the best tool for it's intended purpose. I use it for continuous frame seeking with a buffer ([here](https://github.com/RichieHakim/basic_neural_processing_modules/blob/main/bnpm/video.py#L266)). There are 1.9K projects that use decord as a dependency, which is incredible. 

I'm starting to bump into some serious bugs that require knowledgeable maintainers. Is there a path to active maintenance? Is there something that a layman can help with to achieve this?

https://github.com/ralphtheninja/open-funding#grants
https://github.com/nayafia/lemonade-stand

**Comment by RichieHakim** [comment]
@gaurav274 @zhreshold 
I'm happy to also put anyone in contact with some people who can help try to acquire funds from CZI's grants for research software in computer vision.

The value and utility of this repo is evident in the outstanding issues and huge userbase.

**Comment by gaurav274** [comment]
@RichieHakim, I've got the bandwidth to help maintain this project. However, I've been unable to contact @zhreshold. I'm not sure how we can move forward.

**Comment by RichieHakim** [comment]
@gaurav274 (@zhreshold)
1) Maybe ping his linkedin? 
2) The dmlc governance / structure seems to have disappeared (at least publicly). It has sort of drifted into a distributed group of people maintaining xgboost and and dgl. I've been using your eva-decord repo since mac-12.0.

**Comment by gaurav274** [comment]
@RichieHakim Unfortunately, I haven't heard back from @zhreshold. I tried reaching out via LinkedIn as well. Does it make sense to create a fork and maintain that?

**Comment by RichieHakim** [comment]
Yes, I think it's safe to assume @zhreshold has moved on. 
I'm throwing out some hail mary messages in slacks and emails, but I'm not very optimistic about them. I also took the liberty of CC'ing you and making some assumptions, sorry if that I presumed incorrectly. I'll try a bit longer to get in touch with DMLC organizers so that you (and maybe I) can be made a maintainer. That would be best. If there is no movement on that, then yes a fork will be necessary. Maybe give it a week or two?
